.vscode/settings.json

```diff
@@ -12,6 +12,7 @@
         "*.c": "c",
         "*.cpp": "cpp",
         "*.hpp": "cpp",
-        "xstddef": "c"
+        "xstddef": "c",
+        "type_traits": "c"
```

Comment:
@jackhumbert or @skullydazed this okay?

Reply:
This little guy always makes it into this file. Not sure what it is about my setup that does this. But it doesn't seem to be used by others. Removed line in https://github.com/qmk/qmk_firmware/pull/4703/commits/5184a401d2f8250ab48d7d96c65d28079716e651